lib/node/fullnode.js

```diff
@@ -603,6 +628,18 @@ class FullNode extends Node {
 
     return false;
   }
+
+  /**
+   * Retrieve compact filter by hash.
+   * @param {Hash} hash
+   * @returns {Promise} - Returns {@link Buffer}.
+   */
+
+  async getBlockFilter(hash) {
+    assert(this.filterindex, 'Filter indexer not enabled,' +
```

Comment:
It's common to return either `null` or `[]` in the case the filter is not enabled, this how the other indexing methods behave.